src/openrct2-ui/windows/Park.cpp

```diff
@@ -1446,31 +1446,43 @@ static void window_park_objective_invalidate(rct_window* w)
     window_park_anchor_border_widgets(w);
 }
 
-/**
- *
- *  rct2: 0x0066945C
- */
-static void window_park_objective_paint(rct_window* w, rct_drawpixelinfo* dpi)
+static void window_park_objective_bars_paint(
+    float percentage, ScreenCoordsXY* screenCoords, rct_window* w, rct_drawpixelinfo* dpi, int32_t colour)
 {
-    WindowDrawWidgets(w, dpi);
-    window_park_draw_tab_images(dpi, w);
+    if (percentage > 1)
+    {
+        percentage = 1;
+    }
```

Comment:
I'm wondering if this wouldn't be more elegant with a `clamp`, to guarantee that it is within boundaries:
```suggestion
    percentage = std::clamp(percentage, 0, 1);
```